backend/cypress/support/cross-validation.js

```diff
@@ -1,8 +1,5 @@
 export function testCrossValidation() {
   cy.url().should('match', /\/audit\/cross-validation\/[0-9A-Z]{17}/);
   cy.get(".usa-button").contains("Begin Validation").click();
-  cy.get('.usa-fieldset').contains('Looks good!');
-
-  const reportTestId = '2023AUG0001000010';
-  cy.visit(`/audit/submission-progress/${reportTestId}`);
```

Comment:
I had done this because the other `test*` methods seem to click `Continue` to return to the main submission page, but the cross validation page doesn't have such a link. I feel like we should still try to return to that page here, though, to match the others. Thoughts?

Reply:
Makes sense! I've extended the test to walk through locking the submission for certification, which then sends it back to the checklist view.